packages/patternfly-4/react-core/src/components/Modal/ModalBoxBody.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/ModalBox/modal-box.css';
+
+export interface ModalBoxBodyProps extends React.HTMLProps<HTMLDivElement> {
+  /** content rendered inside the ModalBoxBody */
+  children?: React.ReactNode;
+  /** additional classes added to the ModalBoxBody */
+  className?: string;
```

Comment:
Where did
```
/** id of the ModalBoxBody */
  id: PropTypes.string.isRequired
```
go?

Reply:
I didn't see a reason for it to be required, and id is already on `HTMLProps<HTMLDivElement>`